test/support/helpers/review-return-result.helper.js

```diff
@@ -0,0 +1,87 @@
+'use strict'
+
+/**
+ * @module ReviewReturnResultHelper
+ */
+
+const { randomInteger } = require('./general.helper.js')
+const { generateUUID } = require('../../../app/lib/general.lib.js')
+const { generateLicenceRef } = require('./water/licence.helper.js')
+const { generateReturnLogId } = require('./return-log.helper.js')
+const ReviewReturnResultModel = require('../../../app/models/review-return-result.model.js')
+
+/**
+ * Add a new return result for 2pt matching
+ *
+ * If no `data` is provided, default values will be used. These are
+ *
+ * - `id` - [random UUID]
+ * - `returnId` - v1:1:[the generated licenceRef]:[the generated returnRequirement]:2022-04-01:2023-03-31
+ * - `returnReference` - `10031343`
+ * - startDate - 2022-04-01
+ * - endDate - 2022-05-06
+ * - dueDate - 2022-06-03
+ * - receivedDate - 2022-06-03
+ * - status - `completed`
+ * - underQuery - false
+ * - nilReturn - false
+ * - description - `Lands at Mosshayne Farm, Exeter & Broadclyst`
+ * - purposes - {}
+ * - quantity - 0
+ * - allocated - 0
+ * - abstractionOutsidePeriod - false
+ * - updatedAt - new Date()
+ *
```

Comment:
The `created_at` and `updated_at` get automatically populated when the record is created. So, we shouldn't need to default them.